src/App.js

```diff
@@ -40,8 +41,17 @@ class App extends Component {
 
   componentDidMount() {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-    .then((response) => response.json())
+    .then((response) => {
+      if(!response.ok) {
+        this.setState({error: 'server error'})
+        throw Error(response.status)
+      } else {
+        return response.json()
+      }
+    })
     .then((data) => this.setState({movies: data.movies, individualMovie: null}))
+    .catch(error => alert(`error at ${error}`))
```

Comment:
Assigning 'null' to `individualMovie` might create some bugs in your code. The best practice is always to assign the data type that you're expecting to receive from the API. Is `individualMovie` an object, array, number, or string?